addons/binding/org.openhab.binding.windcentrale/examples.txt

```diff
@@ -0,0 +1,124 @@
+
```

Comment:
It is only as reference. It is not used in any way in the binding.
Main purpose for me is that when the windcentrale protocol/API is changing, I know the 'from' situation.